internal/action/setup.go

```diff
@@ -30,21 +31,27 @@ func (s *Action) Setup(c *cli.Context) error {
 	out.Printf(ctx, "🌟 Welcome to gopass!")
 	out.Printf(ctx, "🌟 Initializing a new password store ...")
 
-	if name := termio.DetectName(c.Context, c); name != "" {
+	if name := termio.DetectName(ctx, c); name != "" {
 		ctx = ctxutil.WithUsername(ctx, name)
 	}
-	if email := termio.DetectEmail(c.Context, c); email != "" {
+	if email := termio.DetectEmail(ctx, c); email != "" {
 		ctx = ctxutil.WithEmail(ctx, email)
 	}
+
+	// age: only native keys
+	ctx = age.WithOnlyNative(ctx, true)
```

Comment:
Because it's not recommended and I was considering to drop support for these entirely.

But then I did like the possible usecase with encrypting for github recipients too much that I didn't remove it, yet.
But at least as the primary "owner" key I think we should require native keys. Especially since they are trivial to generate / manage and it's recommended to have per application keys anyway.